packages/gatsby/src/schema/infer/inference-metadata.js

```diff
@@ -115,7 +119,7 @@ const getType = (value, key) => {
 }
 
 const updateValueDescriptor = (
-  { nodeId, key, value, operation = `add` /*: add | del*/, descriptor = {} },
```

Comment:
Fwiw, the colon in `/*: add | del */` was a workaround for Flow which would support it as if it weren't a comment. This way you could add types to a js document without having to worry about post processing / scrubbing types. Best of both worlds.

So the colon here (I think I've added it myself) is not a typo...

Reply:
Yeah but our eslint config complains about this syntax for me (in Webstorm)

Reply:
Oh hm, my webstorm did not. Alright well nevermind then :)

Reply:
Randomly looking at a TS wip I realized the error is that the strings should have been quoted :p

I just don't have the flow package installed or enabled for my webstorm so I'm not getting the complaints, I guess.